src/config/mnemonic.ts

```diff
@@ -3,6 +3,7 @@ import path from 'path'
 import { ConfigExtender, HttpNetworkConfig } from '@nomiclabs/buidler/types'
 import { aragenMnemonic } from '../params'
 import { readJsonIfExists } from '../utils/fsUtils'
+import { isHexString } from 'ethers/utils'
```

Comment:
Nitpick: we usually move external imports to be ahead of local imports (easier to see which are in our control / within the same context)

Reply:
Fixed by https://github.com/aragon/buidler-aragon/pull/115